app/src/main/java/org/fossasia/openevent/general/social/SocialLinksFragment.kt

```diff
@@ -80,7 +80,15 @@ class SocialLinksFragment : Fragment() {
                 Snackbar.make(socialLinksCoordinatorLayout, it, Snackbar.LENGTH_LONG).show()
             })
 
-        loadSocialLink()
+        socialLinksViewModel.internetError
+            .nonNull()
+            .observe(viewLifecycleOwner, Observer {
+                if (it) {
+                    rootView.socialNoInternet.visibility = View.VISIBLE
```

Comment:
rootView.socialNoInternet.isVisible = it

Reply:
done..